src/components/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <p>Page is under construction!</p>
+);
```

Comment:
- [ ] Please replace this paragraph with a button "Check status" as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/199720987-7ffc0740-fa65-449d-ac54-f0ec0f9d3cac.png)